_data/sponsors.yml

```diff
@@ -0,0 +1,45 @@
+#
+# Data for the sponsors display on the home page.
+# Additional data like description can be used on a dedicated sponsors page in # the future
+#
+# Bloomberg is commented out until we get approval sorted.
+# - name: Bloomberg
+#   anchor: bloomberg
+#   position: 3
+#   logo: /assets/images/logos/bloomberg.png
+#   logosmall: /assets/images/logos/bloomberg-small.png
+#   logobalanced: /assets/images/logos/bloomberg-balanced.png
+#   description: |
+#     tbd
+#   links:
+#     - urltext: View the Bloomberg website
+#       url: https://www.bloomberg.com/
+- name: GitHub
+  highlight: false
+  position: 2
+  anchor: github
+  logo: /assets/images/logos/github.png
+  logosmall: /assets/images/logos/github-small.png
+  logobalanced: /assets/images/logos/github-balanced.png
+  description: |
+    GitHub is used for source code hosting, issue tracking, continuous
+    integration and other aspects of running the project. GitHub is allocating
+    GitHub Actions compute resources to Trino CI beyond the volume available to
+    smaller open source projects.
```

Comment:
We don't need to include an explanation of how they are providing sponsorship.

Reply:
This is not user visible at this stage. Only in the source and can be used later.. should I completely remove it for now?

Reply:
Removed now.